networkx/linalg/attrmatrix.py

```diff
@@ -169,28 +169,29 @@ def attr_matrix(
     G : graph
         The NetworkX graph used to construct the attribute matrix.
 
-    edge_attr : str, optional
+    edge_attr : str, optional (default: number of edges for each matrix element)
```

Comment:
Same here regarding using "default" to indicate default _behavior_ and not a default _value_.

Also, `edge_attr` and `node_attr` can be callables too, but this isn't mentioned here.